pontoon/urls.py

```diff
@@ -97,6 +97,7 @@
     url(r"^translations/", include("pontoon.translations.urls")),
     url(r"", include("pontoon.teams.urls")),
     url(r"", include("pontoon.tour.urls")),
+    url(r"", include("pontoon.terminology.urls")),
```

Comment:
Nit: We haven't done that much so far but I'm a fan of namespaces in URLs. So I'd advocate for putting this in a `^terminology/` namespace, thus making the get terms view called via `/terminology/get-terms/`. I think it makes things clearer, and also reduces the chances of conflicts in URL paths.